lib/utils/constant.dart

```diff
@@ -1,6 +1,7 @@
-const double hundred = 100;
-const String imageURL = 'https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/index.png';
+const String imageURL = 'https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/home/index.png';
```

Comment:
do you still need this one? 🙂

Reply:
yes this is the url for the image of the pokemon

Reply:
aight 👍